deploy/docker-compose/README.md

```diff
@@ -0,0 +1,261 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between the local development of MAPs in a laptop using the MONAI Deploy App SDK and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test/research PACS, for further validation, confidently taking steps towards production.   
+
+Reusing the same essential core services for DICOM I/O and AI workflow orchestration provides the same functionality and consistent experience independently of where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
+
+- [Docker Engine](https://docs.docker.com/engine/install/) v20.10.18+
+- [Docker Compose Plug-in](https://docs.docker.com/compose/install/) v2.10.2+
+- MONAI Deploy MAPs built using the [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk)
+- [jq](https://stedolan.github.io/jq/)
+- (Optional) [Informatics Gateway CLI](https://github.com/Project-MONAI/monai-deploy-informatics-gateway/releases)
+
+## Installation
+
+To use MONAI Deploy Lite, install all prerequisites & download this entire directory from [GitHub](https://github.com/Project-MONAI/monai-deploy/tree/main/docker-compose/deploy/docker-compos). 
+
```

Comment:
This link does not work,404 not found. 
```
git clone https://github.com/Project-MONAI/monai-deploy/tree/main/docker-compose/deploy/docker-compos
Cloning into 'docker-compos'...
fatal: repository 'https://github.com/Project-MONAI/monai-deploy/tree/main/docker-compose/deploy/docker-compos/' not found
```

Reply:
Also please add instructions to delete the installation.

Reply:
That's expected as this PR is still open.😂

Reply:
Also can you please add instructions to delete or stop the installation